agent/engine/task_manager.go

```diff
@@ -422,11 +428,6 @@ func (mtask *managedTask) handleContainerChange(containerChange dockerContainerC
 		}
 	}
 
-	// Update the container health status
-	if container.HealthStatusShouldBeReported() {
```

Comment:
Could you explain a bit why this block of code is deleted?

Reply:
+1. i don't think this should be deleted?

Reply:
**updateContainerMetadata** already has this part of logic.